File: CP#2-TEXQuotes/Solution.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        int c = 0;
        Scanner scan = new Scanner(System.in);
        
        while (scan.hasNext()) {
            String str = scan.nextLine();
            for (int i = 0; i < str.length(); i++) {
                if (str.charAt(i) == '\"') {
                    if (c == 0) {
                        System.out.print("``");
                        c = 1;
                        continue;
                    }
                    else {
                        System.out.print("''");
                        c = 0;
                        continue;
                    }
                }
                System.out.print(str.charAt(i));
            }
            // System.out.println("\n");
        }
    }
}

File: CP#2-TEXQuotes/Solution.py
c = 0
while True:
    n = input()
    a = []
    for i in n:
        if i == "\"":
            if c == 0:
                a.append("``")
                c = 1
            else:
                a.append("''")
                c = 0
        else:
            a.append(i)
    print("".join(a))